lib/resque.rb

```diff
@@ -111,6 +111,26 @@ def after_fork=(after_fork)
     @after_fork = after_fork
   end
 
+  # Runs after a job is performed
+  def after_perform(&block)
+    block ? (@after_perform = block) : @after_perform
+  end
+
+  # Set the after_perform proc.
+  def after_perform=(after_perform)
```

Comment:
Plugins can declare `after_perform` callbacks, but this is new.

TODO: verify the semantics of this block match the plugin callback version (arguments / exception behavior)